zwave_js/rootfs/etc/cont-init.d/config.sh

```diff
@@ -14,7 +14,9 @@ if bashio::config.has_value 'network_key'; then
     # we don't know which one to pick so we have to exit. If they are both set
     # and do match, we don't need to do anything
     if bashio::config.has_value 's0_legacy_key'; then
-        if bashio::config.equals 's0_legacy_key' "$(bashio::config \"network_key\")"; then
+        network_key=$(bashio::string.upper "$(bashio::config 'network_key')")
+        s0_legacy_key=$(bashio::string.upper "$(bashio::config 's0_legacy_key')")
```

Comment:
We declare `network_key` at the top of the file. Is that something we need to do for `s0_legacy_key` too? I only know minimal bash, so just asking about things I see.

Reply:
I am not actually sure what the `declare` does. In Googling around it seems optional, but I agree that either we should do it for all or do it for none. It seems to be a pattern to do this in other addons, so I will add the missing declarations